timefold-solver-python-core/src/main/python/score/_score_analysis.py

```diff
@@ -541,6 +595,77 @@ def constraint_analyses(self) -> list[ConstraintAnalysis]:
                 list['_JavaConstraintAnalysis[Score]'], self._delegate.constraintAnalyses())
         ]
 
+    @dispatch(str, str)
+    def constraint_analysis(self, constraint_package: str, constraint_name: str) -> ConstraintAnalysis:
+        """
+        Performs a lookup on `constraint_map`.
+
+        Parameters
+        ----------
+        constraint_package : str
+        constraint_name : str
+
+        Returns
+        -------
+        ConstraintAnalysis
+            None if no constraint matches of such constraint are present
+        """
+        return ConstraintAnalysis(self._delegate.getConstraintAnalysis(constraint_package, constraint_name))
+
+    @dispatch(ConstraintRef)
+    def constraint_analysis(self, constraint_ref: 'ConstraintRef') -> ConstraintAnalysis:
+        """
+        Performs a lookup on `constraint_map`.
+
+        Parameters
+        ----------
+        constraint_ref : ConstraintRef
+
+        Returns
+        -------
+        ConstraintAnalysis
+            None if no constraint matches of such constraint are present
+        """
+        return ConstraintAnalysis(self._delegate.getConstraintAnalysis(constraint_ref._to_java()))
+
+    @property
+    def summarize(self) -> str:
```

Comment:
`summarize` a method name not an attribute name. Use `summary` if used as an attribute.

Reply:
I agree - `summarize` is a method name, not a property. This should not be a property.